src/library/scala/collection/mutable/PriorityQueue.scala

```diff
@@ -15,6 +15,12 @@ import generic._
 /** This class implements priority queues using a heap.
  *  To prioritize elements of type A there must be an implicit
  *  Ordering[A] available at creation.
+ * 
+ *  If multiple elements have the same priority in the ordering of this
+ *  PriorityQueue, no guarantees are made regarding the order in which elements
+ *  are returned by `dequeue` or `dequeueAll`. In particular, that means this
+ *  class does not guarantee first in first out behaviour that may be
```

Comment:
- I would capitalise _first in first out_, and maybe connect the words with dashes, and add an acronym afterwards. E.g:
> "First-In-First-Out (FIFO) behaviour. 

- I would replace "behaviour that may be incorrectly inferred from the Queue part of the name of this class."  with the following:  
> behaviour, contrary to what you may expect from a class named "Queue".